cmake/PythonFixUpOnMac.cmake

```diff
@@ -46,7 +46,9 @@ execute_process(COMMAND "chmod" "+w" "${LOCAL_PYTHON_LIBRARY}")
 execute_process(COMMAND "install_name_tool" -id "@executable_path/${PYTHON_LIB_FILENAME}" "${LOCAL_PYTHON_LIBRARY}")
 
 # for the energyplus executable, just find the python dynamic library right next to it for sure
-get_prerequisites("${EXECUTABLE_PATH}" PREREQUISITES 1 1 "" "")
+get_filename_component(exepath ${EXECUTABLE_PATH} DIRECTORY)
+#  <exepath> is the path to the top level executable used for @executable_path replacement on the Mac. We must pass it to silence a warning
+get_prerequisites("${EXECUTABLE_PATH}" PREREQUISITES 1 1 ${exepath} "")
```

Comment:
```
GET_PREREQUISITES(<target> <prerequisites_var> <exclude_system> <recurse>
                  <exepath> <dirs> [<rpaths>])
```

https://cmake.org/cmake/help/latest/module/GetPrerequisites.html

Interestingly, TIL `GetPrerequisites` is deprecated since cmake 3.16